src/wrappers/any.jl

```diff
@@ -144,7 +144,11 @@ end
 
 # XXX: Need to convert julia vectors to StdVector. This function helps us avoid having
 # CxxWrap as a direct dependency in Ray.jl
-_submit_task(dir, fd, oids::AbstractVector) = _submit_task(dir, fd, StdVector(oids))
+function _submit_task(dir, fd, oids::AbstractVector)
+    # https://github.com/JuliaInterop/CxxWrap.jl/issues/367
+    args = isempty(oids) ? StdVector{ObjectID}() : StdVector(oids)
```

Comment:
ideally `StdVector(ObjectID[])` would be equivalent to `StdVector{ObjectID}()`

Reply:
I was going to advocate that we just pass in a `ObjectID[]` given your statement above but what you intended to convey is that these `StdVector(ObjectID[])` and `StdVector{ObjectID}()` are not the same even though they are equal:
```julia
julia> StdVector(ObjectID[])
0-element CxxWrap.StdLib.StdVectorAllocated{Any}

julia> StdVector{ObjectID}()
0-element CxxWrap.StdLib.StdVectorAllocated{ObjectID}

julia> StdVector(ObjectID[]) == StdVector{ObjectID}()
true
```

Reply:
yes! I didn't actually check equality though (assumed it would fail). But since they have different parameter types the first one doesn't get dispatched to the Cxx `_submit_task`.